controlflowintro.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Control Flow
+---
+
+# Control Flow
+
+Control flow concerns how programs proceed in their execution to arrive a desired outcome. Often times, we write programs to make decisions based on certain **conditions** such as input from the user or values in the data. Different conditions lead to a variety scenarios. Programs can make choices concerning meteorological conditions such as wind velocity. Consider a Python program that draws wind barbs on a map. It will have to take actions depending on the velocity of the wind data. The program will have to draw a flag for winds of greater than 47 knots but only one barb for 10 knot winds.
+
+In addition, control flow can also concern a program proceeding through a list of values or while some condition has not yet been met. A rawinsonde may send data until a certain pressure level has been attained, for example.
```

Comment:
"or while some condition has not yet been"--maybe before this add something like "or continuing to execute a set of statements"